lib/extensions.js

```diff
@@ -63,6 +64,8 @@ function getBinaryName() {
     binaryName = flags['--sass-binary-name'];
   } else if (process.env.SASS_BINARY_NAME) {
     binaryName = process.env.SASS_BINARY_NAME;
+  } else if (process.env.npm_config_sass_binary_name) {
+    binaryName = process.env.npm_config_sass_binary_name;
```

Comment:
I don't think these `npm_config_` work as expected. You should use `npmconf` instead.